x-pack/otel/processor/beatprocessor/README.md

```diff
@@ -7,18 +7,22 @@
 [development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
 
 > [!NOTE]
-> This component is currently in development and no functionality is implemented.
-> Including it in a pipeline is a no-op.
-> The documentation describes the intended state after the functionality is implemented.
+> This component is currently in development and functionality is limited.
 
 The Beat processor (`beat`) is an OpenTelemetry Collector processor that wraps the [Beat processors].
-This allows you to use Beat processorss like e.g. [add_host_metadata] anywhere in the OpenTelemetry Collector's pipeline, independently of Beat receivers.
+This allows you to use Beat processors like e.g. [add_host_metadata] anywhere in the OpenTelemetry Collector's pipeline, independently of Beat receivers.
 
 > [!NOTE]
 > This component is only expected to work correctly with data from the Beat receivers: [Filebeat receiver], [Metricbeat receiver].
 > This is because it relies on the specific structure of telemetry emitted by those components.
 > Using it with data coming from other components is not recommended and may result in unexpected behavior.
 
+The processor enriches the telemetry with host metadata by using the [add_host_metadata] processor under the hood.
+Note that configuration is limited at this stage.
+Host metadata is added unconditionally and cannot be disabled.
+You can configure the host metadata enrichment using the options that the [add_host_metadata] processor allows.
```

Comment:
There are a couple processors enabled by default for beats receivers: add_host_metadata, add_cloud_metadata, add_docker_metadata, add_kubernetes_metadata. I believe these can't be disabled unless you also customize the processors list inside of the beat receiver config. Should this be mentioned here?

Reply:
Oh right, this is a very good point. The docs should mention this.

When the OTel Beat processor is complete, shouldn't the default processors in Beat receivers be disabled?

Reply:
Hmmm, I'm not sure. The default behavior of the beats receivers was to mimic standard beats. I added this point to our meeting today so we can discuss with the team.

Reply:
Added the docs here: https://github.com/elastic/beats/pull/46112/commits/eed6f65b0ddd07ad38300e1459ea4e4d5581b278